ambu

```diff
@@ -21,8 +21,18 @@ if [ ! -d "$CURRENTDIR" ]; then
     exit 1
 fi
 
-mkdir -p $BUILDDIR
+command -v ctags >/dev/null 2>&1 || {
+        echo >&2 "ctags required for building your project. Please install it!"
+        exit 1
+}
+
 ctags --language-force=c --exclude=$PROJECTNAME *.ino
+if ! grep -q "$(printf '_TAG_PROGRAM_NAME\tExuberant Ctags\t//')" tags; then
+    echo "ctags generated output not compatible with ambu. Please reinstall it!"
```

Comment:
@Adman I am not convinced reinstalling ctags would help here. Would it make more sense to provide information on how to install `ctags` that ambu expects?